main.tf

```diff
@@ -7,6 +7,7 @@ resource "aws_cloudtrail" "cloudtrail" {
   s3_bucket_name        = aws_s3_bucket.cloudtrail_bucket.id
 
   enable_log_file_validation = var.enable_cloudtrail_log_file_validation
+  kms_key_id                 = aws_kms_key.cloudtrail_bucket_encryption_key.arn
```

Comment:
Because we're adding encryption to the CloudTrail, the comment @ the top of this file should change specifying that we are adding encryption, just not with AWS owned keys.

Reply:
comment deleted.